osidb/models.py

```diff
@@ -1193,6 +1177,17 @@ class Meta:
     def __str__(self):
         return str(self.uuid)
 
+    def _validate_tracker_flaw_accesses(self):
+        """
+        Check whether an public tracker is associated with an embargoed flaw.
+        """
+        if not self.is_embargoed and Flaw.objects.filter(
```

Comment:
I would call `exists()` here after the `filter()` call, functionally it's the same but we need to be careful with performance as we're making expensive joins

Reply:
makes sense, called exists after filtering.